src/olympia/templates/photon-footer.html

```diff
@@ -14,7 +14,7 @@ <h4 class="Footer-links-header"><a href="{{ url('home') }}">Add-ons</a></h4>
                 <li><a href="{{ url('devhub.index') }}">Developer Hub</a></li>
                 <li><a href="https://developer.mozilla.org/docs/Mozilla/Add-ons/AMO/Policy">Developer Policies</a></li>
                 <li><a href="https://discourse.mozilla-community.org/c/add-ons">Forum</a></li>
-                <li><a class="Footer-bug-report-link" href="https://developer.mozilla.org/docs/Mozilla/Add-ons/Contact_us">Report a bug</a></li>
+                <li><a class="Footer-bug-report-link" href="https://addons.mozilla.org/en-US/about">Report a bug</a></li>
```

Comment:
Could you please remove the locale from this link?

```
https://addons.mozilla.org/about
```

Reply:
I did a stupid mistake of deleting the fork, I have cloned it again.
Can you please guide me further?

Reply:
Should I create a fresh PR?

Reply:
Not necessarily. You can fetch the current patch locally:

```
curl https://github.com/mozilla/addons-server/pull/17634/commits/f6f962da236c85c0b6bf0508b108f486260c473f.patch | git am -
```

Reply:
I ran the command.
It is showing patch is empty

Reply:
mm, running `git show` should show you this patch. Is it not what you have?

Reply:
It is not showing me this patch, it showing me the patch I did an hour ago to this repo.

Reply:
heh ok, well in this case I suggest you start from scratch given that it is only 1 url to update.

Reply:
Yes sure I will do that xD